include/nbl/asset/metadata/CGLTFPipelineMetadata.h

```diff
@@ -6,7 +6,9 @@
 #include "nbl/asset/ICPUDescriptorSet.h"
 #include "nbl/asset/ICPUPipelineLayout.h"
 
-namespace nbl::asset
+namespace nbl
+{
+namespace asset
 {
```

Comment:
?

Reply:
though it was the reason for weirds errors, redo!

Reply:
there was probably a double `}}` there and the end

Reply:
it is fine with `nbl::asset`, got redefinitions due to circular dependency

Reply:
didi it myself